functions/memorystore/redis/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "nodejs-docs-samples-functions-memorystore-redis",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=8.13.0"
+  },
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha test/*.test.js --timeout=20000 --exit"
+  },
+  "dependencies": {
+    "redis": "^2.8.0"
+  },
+  "devDependencies": {
+    "@google-cloud/nodejs-repo-tools": "^3.3.0",
+    "mocha": "^6.1.4",
+    "proxyquire": "^2.1.0",
+    "sinon": "^7.3.2"
+  },
+  "cloud-repo-tools": {
```

Comment:
@fhinkel is this still relevant if we plan to nix `nodejs-repo-tools`?

(It's useful documentation of what env vars we need - but perhaps there's a better place to pit that?)

Reply:
If you don't use `repo-tools` in the tests, you can delete that dependency and the `cloud-repo-tool` field. Thanks!

Reply:
Done :+1: